src/Toggle/MysqlActivatedToggleGateway.php

```diff
@@ -43,20 +62,21 @@ public function __construct( Connection $connection, IsToggleActive $toggleCheck
      */
     public function getAllMyActivatedToggles( $userIdentifier )
```

Comment:
Imagine a presentation layer for an API. It doesn't care about the gateways, it just wants to call UseCases. Where does the API tell the UseCase what the current User is?